PROTOCOL.md

```diff
@@ -802,6 +804,12 @@ The following is an example for the column definition of a table that leverages
   }
 ```
 
+## Usage Tracking
```

Comment:
Let's write this out in full as "Column Mapping Usage Tracking". Especially in git reviews it's hard to see that this is a sub-item of Column Mapping.

Also, shouldn't we cover the requirements for Column Mapping itself first? This interleaving of two features is confusing to me. We should probably move this entire section down below section "Reader Requirements for Column Mapping". The Writer Requirements for Column Mapping then have a forward reference to this feature, but we can make that work by annotation that reference with a simple "(see below)".

Reply:
Thank you for the great suggestions!